LEAF_Request_Portal/admin/index.php

```diff
@@ -492,6 +492,7 @@ function hasDevConsoleAccess($login, $db_phonebook)
 $t_menu->assign('action', $action);
 $t_menu->assign('orgchartPath', Config::$orgchartPath);
 $t_menu->assign('name', XSSHelpers::sanitizeHTML($login->getName()));
+$t_menu->assign('siteType', XSSHelpers::xscrub($settings['siteType']));
```

Comment:
Make sure PR#109 goes in first before this and you'll have to do a pull/merge so this doesnt result in a conflict.

Reply:
removed it from this PR